unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/AvatarSlot/NftTypeColorSupportingSO.cs

```diff
@@ -0,0 +1,17 @@
+﻿using UnityEngine;
+
+[CreateAssetMenu(fileName = "NftTypeColorSupporting", menuName = "Variables/NftTypeColorSupporting")]
+public class NftTypeColorSupportingSO : ScriptableObject
+{
+    [SerializeField] public SerializableKeyValuePair<string, bool>[] nftTypes;
```

Comment:
its not clear to me what is the type of the nft? the category? perhaps a rename?

Reply:
You're right. In order to make it a bit more intuitive, I have renamed the variable to `colorSupportingByNftType` (https://github.com/decentraland/unity-renderer/pull/5028/commits/9d1f7544dfc7c5a924ce4532ccf3c6e24fff85ef)